api/ImageData.json

```
@@ -148,6 +148,68 @@
                 "deprecated": false
               }
             }
+          },
+          "pixelFormat_option": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
+              "support": {
+                "chrome": {
+                  "version_added": "137"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "[dataArray_parameter_]accepts_Float16Array": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
+              "support": {
+                "chrome": {
+                  "version_added": "137"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
```

Given that the new `pixelFormat` setting reflects `Float16Array` support, we can remove this "accepts" subfeature.

```suggestion
          }
```